### PR TITLE
Exposed bPaused to Lua scripting, updated documentation

### PR DESCRIPTION
**Pull request by AlderStoresWordsOfMagic**
The variable `bPaused`, part of `CApp`, is now exposed to Lua scripting. See the documentation for further details.

### CODE REVIEW

File: lua/modules/hyperspace.i [hunk 1]
```diff
@@ -580,9 +580,9 @@ playerVariableType playerVariables;
 %rename("%s") CommandGui::outOfFuel;
 %immutable CommandGui::outOfFuel;
 %rename("%s") CommandGui::bPaused;
-%immutable CommandGui::bPaused;
+// %immutable CommandGui::bPaused;
```
Comment (ChronoVortex) [inline lua/modules/hyperspace.i:583]: This doesn't need to be commented, it can be fully removed.

File: lua/modules/hyperspace.i [hunk 2]
```diff
@@ -580,9 +580,9 @@ playerVariableType playerVariables;
...
 %rename("%s") CommandGui::bAutoPaused;
-%immutable CommandGui::bAutoPaused;
+%immutable CommandGui::bAutoPaused; // potentially find a way to try ESC pausing, currently does nothing
```
Comment (ChronoVortex) [inline lua/modules/hyperspace.i:585]: This comment didn't need to be removed.

**Review by ChronoVortex — CHANGES_REQUESTED** [review]

File: lua/modules/hyperspace.i (hunk 1 above)
Comment (AlderStoresWordsOfMagic) [reply lua/modules/hyperspace.i:583]: alrighty!

File: lua/modules/hyperspace.i (hunk 2 above)
Comment (AlderStoresWordsOfMagic) [reply lua/modules/hyperspace.i:585]: i don't remember removing it?

**Review by ChronoVortex — APPROVED** [review]

**Pull request merged**